sponsorship/src/main/java/com/kyriba/conference/sponsorship/service/PlanService.java

```diff
@@ -0,0 +1,45 @@
+package com.kyriba.conference.sponsorship.service;
+
+import com.kyriba.conference.sponsorship.api.dto.PlanRegistrationRequest;
+import com.kyriba.conference.sponsorship.domain.Plan;
+import com.kyriba.conference.sponsorship.domain.PlanCategory;
+import org.springframework.stereotype.Service;
+
+import java.util.Collection;
+import java.util.Map;
+
+
+/**
+ * @author M-ASL
+ * @since v1.0
+ */
+@Service
+public class PlanService

```

Comment:
Service is not needed at that stage.

Reply:
I believe, it is similar to https://github.com/gariklolkin/conference/pull/10#discussion_r286013126, so for now, I'd prefer to leave it as is